.travis.yml

```diff
@@ -45,7 +45,7 @@ script:
   - bundle exec rake portus:assets:compile
 
   # Ruby tests
-  - bundle exec rspec spec packaging/suse/portusctl/spec
+  - bundle exec rspec spec packaging/suse/portusctl/spec --tag ~integration
```

Comment:
Why are we dropping the integration tests?

Reply:
In Travis we dropped it anyways because it's been broken for a while. I plan to re-write the integration tests in a similar way to what I did in [portusctl](https://github.com/openSUSE/portusctl), because it's way more reliable and it's easier to write tests.